JapaneseFilter/sections/specific.txt

```diff
@@ -5,6 +5,31 @@
 !
 ||feed.mikle.com^$domain=blog.livedoor.jp|crx7601.com|fate-rpg.jp|gadgetlife2ch.blomaga.jp|gtoys.blog.fc2.com|h1g.jp|hdouga.com|ibarakinews.jp|obutu.com|totalwar.doorblog.jp|xn--bckadddb2a0d4dnc8dwhngdtb58aya0l4a2om843g5npczp7dwh5g.com|xn--cckea5a6cidcbh6ce7ghug17a2ge3aht3nwigef51658aw7kd.com
 !
+renote.jp##[class$="__ad__container"]
+||gundam-futab.info/wp-content/themes/matome/rss/rss.html
+gundam-futab.info###sidebar-widget > .widget_pc_text:first-child
+gundam-futab.info##.rss_label
+gundam-futab.info##.sousyoku
+gundam-futab.info##.widget-under-article
+||bblog.sso.biglobe.ne.jp/js/insert-ad.
+||jav.guru^*/1600.gif
+||jin115.com/js/amazon_
+||jin115.com/rss*$subdocument,~third-party
+jin115.com###side_left
+jin115.com###side_right > .plugin-memo:first-child
+jin115.com###side_right > .plugin-memo:last-child
+jin115.com##.article_bodymore > table[style^="text-align:left;"]
+jin115.com##div[style="width:728px; height:90px;"]
+||pinknotora.net/468-73.jpg
+||pinknotora.net/DMM.php
+||pinknotora.net/rsp.htm
+||pinkt.cocolog-nifty.com/pink/antenna.js
+english.evidus.com###sideMenu > font[color="#666666"]
+english.evidus.com###sideMenu > img[src$="_line.gif"]:nth-of-type(-n+2)
```

Comment:
Why not `english.evidus.com###sideMenu > img[width="180"][height="16"]`?

Reply:
Ah, that looks better. Will fix that.

Reply:
Fixed.